### ISSUE TITLE
Number of CPU cores cannot be adjusted if no documents loaded

### ISSUE THREAD

**Issue by Gitoffthelawn** [WinMerge/winmerge#1521]
In WinMerge 2.16.22.0 x64, the number of CPU cores for compare operations cannot be set in the settings UI if no documents are loaded.

**Comment by sdottaka** [comment]
Could you elaborate?

**Comment by Gitoffthelawn** [comment]
Yes. If you load WinMerge without any documents, and then go to the settings UI, the user is unable to adjust the number of CPU cores used by default.

**Comment by sdottaka** [comment]
When you say "The number of CPU cores used", do you mean "Number of compare threads (...)" in the Options dialog?
When you say that the value cannot be adjusted, do you mean that you cannot enter a number?
Or do you mean something else?
I don't understand if this issue is a bug or a feature request.

**Comment by Gitoffthelawn** [comment]
> When you say "The number of CPU cores used", do you mean "Number of compare threads (...)" in the Options dialog?

Yes.  Sorry, I should have used the current English translation.  I'm actually using another language right now.

> When you say that the value cannot be adjusted, do you mean that you cannot enter a number?

Correct.  The field is dimmed (no input allowed).

> I don't understand if this issue is a bug or a feature request.

Bug, IMO. :)

**Comment by sdottaka** [comment]
Oh, I'm sorry. I think I understand now.
I think you probably chose BinaryContents or Modified Date and Size as the Compare method.
The above compare method uses only one CPU core because it is slower if multiple CPU cores are used.
Therefore, we have disabled input in the text box to show this.

**Comment by Gitoffthelawn** [comment]
No problem.

Yes, I have BinaryContents enabled.  What's a little strange is that I've had BinaryContents enabled for quite some time (months), and yet somehow I came across the "Number of compare threads" editable.

Perhaps I temporarily was using another compare method and noticed the issue with the UI for that option begin confusing.

Would there be any other reason why the "Number of compare threads" field would be enabled?
